File: 8_kyu/Simple_Fun_1_Seats_in_Theater.jl
module Solution
  export seatsintheater
  
  """
    Counts the number of seats right behind seat col,row.
    ncols, nrows are the number of seat columns and seat rows.
    Seat position (col, row) starts with (1,1).
  """
  seatsintheater(ncols::Integer, nrows::Integer, col::Integer, row::Integer) = (nrows-row)*(ncols-col+1)
end

File: 8_kyu/Simple_Fun_1_Seats_in_Theater.py
def seats_in_theater(tot_cols, tot_rows, col, row):
    """
    Calculates number of seats strictly+left behind
    tot_cols -- number of seat columns
    tot_rows -- number of seat rows
    col -- seat column position, 1..nCols
    row -- seat row position, 1..nRows
    return -- Number of seats strictly+left behind 
    """
    return (tot_cols+1-col)*(tot_rows-row)